File: livetester-service/src/main/java/com/yorosis/livetester/vo/EnvironmentPresetVO.java
package com.yorosis.livetester.vo;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class EnvironmentPresetVO {
	private long id;
	private long environmentId;
	private String type;
	private String key;
	private BeneficiaryVO beneficiary;
	private ProviderVO provider;
	private PayorVO payor;
	private PaVO paVO;

}

File: livetester-ui/src/shared/vo/environment-preset-vo.ts
import { BeneficiaryVO, ProviderVO, PayorVO } from './claim-vo';

export class PaVO {
    number = '';
    description = '';
}


export class EnvironmentPresetVO {
    id: number;
    environmentId: number;
    type: string;
    key: string;
    beneficiary: BeneficiaryVO = new BeneficiaryVO();
    provider: ProviderVO = new ProviderVO();
    payor: PayorVO = new PayorVO();
    paVO: PaVO = new PaVO();
}
